package.json

```diff
@@ -43,4 +43,4 @@
       "prism.js"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
Please don't include this change in this PR. It is irrelevant to the addition of the support for Io.